listen.py

```diff
@@ -38,7 +38,7 @@ def from_json(cls, j):
             user_name=j.get('user_name', ""),
             timestamp=datetime.utcfromtimestamp(float(j['listened_at'])),
             artist_msid=j['track_metadata']['additional_info'].get('artist_msid'),
-            album_msid=j['track_metadata']['additional_info'].get('album_msid'),
```

Comment:
From a look at the codebase, seems like `album_*` is used in the listenstores (both postgres and influx) and the bigquery schema has columns named `album_*`. Wouldn't be too complex to change it all to `release_*`, although I was thinking maybe there is some reason we moved from `release_*` to `album_*` related to MB? If there isn't, a refactor wouldn't be much effort I think.